### PR TITLE
fix footer when data is not present

### PR DESCRIPTION
**Pull request by ashishmangla0**

### CODE REVIEW

**Comment by ashishmangla0** [comment]
image

**Review by sumitd94 — DISMISSED** [review]

File: src/components/Layout/Layout.module.scss [hunk 1]
```diff
@@ -1,6 +1,9 @@
 .layout {
   margin: 0 auto;
   max-width: 100%;
+  flex-direction: column;
+    display: flex;
+    min-height: 100%;
```
Comment (swarajpure) [inline src/components/Layout/Layout.module.scss:6]: Can we please have the lines properly indented?

**Review by swarajpure — CHANGES_REQUESTED** [review]
Requesting very minor change. Please check.

File: src/components/Layout/Layout.module.scss (hunk 1 above)
Comment (ashishmangla0) [reply src/components/Layout/Layout.module.scss:6]: @swarajpure its fixed

Comment (ashishmangla0) [reply src/components/Layout/Layout.module.scss:6]: image

**Review by swarajpure — APPROVED** [review]
Thank you for the changes! 😄

**Review by swarajpure — APPROVED** [review]
Thank you for the changes! 😄

**Review by swarajpure — APPROVED** [review]
Thank you for the changes! 😄

**Review by whyDontI — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**